tools-local/prebuilt-baseline-online.xml

```diff
@@ -260,6 +260,7 @@
     <Usage Id="NETStandard.Library" Version="2.0.1" />
     <Usage Id="NETStandard.Library" Version="2.0.3" />
     <Usage Id="NETStandard.Library.NETFramework" Version="2.0.1-servicing-26011-01" IsDirectDependency="true" />
+    <Usage Id="Newtonsoft.Json" Version="9.0.1" />
     <Usage Id="Newtonsoft.Json" Version="10.0.3" IsDirectDependency="true" />
     <Usage Id="Newtonsoft.Json" Version="11.0.2" IsDirectDependency="true" />
     <Usage Id="Newtonsoft.Json" Version="12.0.2" IsDirectDependency="true" />
```

Comment:
Newtonsoft.Json 12.0.2 should no longer be prebuilt since we're building it - suggest comparing the generated prebuilt report with yours (bin/prebuilt-report/generated-new-baseline.xml).

Reply:
Yes. It had a suggestion to remove it. I probably had a misconception that 12.0.2 should also be a part of prebuilt. I'll rectify this.